config/binary/packages_pip.txt

```diff
@@ -8,7 +8,7 @@ qpoint
 so3g
 #
 # sotodlib latest tag
-https://github.com/simonsobs/sotodlib/archive/refs/tags/v0.6.8.tar.gz
+https://github.com/simonsobs/sotodlib/archive/refs/tags/v0.6.12.tar.gz
```

Comment:
Why not pull sotodlib from pypi?

Reply:
Good point- this file predates the pypi releases, will change that.